Remove remember merge from request on login

DanielDarrenJones opened the pull request:
Removed the merge on the request when making a login attempt, this resets the remember value which was stopping the remember me cookie from being created.

Fixes: #161

taylorotwell commented:
This doesn't look right at all.

lkoenigsberger commented:
yes agree ;-) 
but throws this error:
```
[ErrorException]
  class_parents(): object or string expected
```
So looks like the route->group needs a string or an object? Did that change in 5.2?

Commit:
Remove remember merge from request on login

Removed the merge on the request on login attempt which resets the remember value, this was stopping the remember me cookie from being created.

Fixes: https://github.com/laravel/spark/issues/161

Commit:
Update LoginController.php
